docs/configuration.md

```diff
@@ -13,6 +13,19 @@ align = more    // For pretty alignment.
 maxColumn = 100 // For my wide 30" display.
 ```
 
+There is a possibility to override scalafmt config for a specific file with
```

Comment:
it's not really file, it's group of lines. i think you can use another comment further down, to modify again.

unfortunately, there's no `off` setting; if you want to restore, you need to set it explicitly.

Reply:
specified it

Reply:
It should be fairly easy to implement some kind of `off` setting to revert to the original settings